.gitignore

```diff
@@ -47,4 +47,11 @@ hack/providers-sync-tools/**/build
 # Build artifacts related to packages
 /build
 /packages/package-values-sha256.yaml
-/packages/**/.imgpkg
\ No newline at end of file
+/packages/**/.imgpkg
```

Comment:
I think we don't generally add IDE-specific files to the repo (you can add them locally).
IFF we do decide to do so, I would suggest just ignoring the .idea directory, rather than all these specific files.

Reply:
moved .idea files to my local global .gitignore